functions/tech-support

```diff
@@ -68,241 +70,256 @@ fi
 export PATH=/sbin:/usr/sbin:$PATH
 
 header Show Tech-Support  
+  
 header CONFIGURATION 
+  
+function show_version { 
+  if cli-shell-api validateTmplPath entitlement; then
```

Comment:
This is one of the cases where cleanup was needed badly. The "entitlement" commands haven't existed in years.

Reply:
I have deleted this block